tests/helpers.js

```diff
@@ -0,0 +1,79 @@
+const VERSIONS_EQUALS = [
```

Comment:
If we instead we opt to keep the current approach and load this `helpers.js` file also in Firefox using `Services.scriptloader.loadSubScript`, then I think we should add an inline comment here to point that out (because it may become surprising at some point, e.g. if trying to use some nodejs API in here and then potentially having an hard time to fully see how that would be makeing the integration test to fail).

Reply:
Note: I am not loading the `helpers.js` file, only `src/index.js`, which was kinda useful to fix #6 but I can live without it :p

Reply:
eh, I looked the integration test too quickly and my brain parsed it wrong :-P, I should have looked more carefully.

I think we can still do it in a pretty simple way using the fact that jest does also provide a jsdom-based fake window without having to do anything special, if I understood correctly what we wanted to test for #6, then testing it in a mocked DOM environment may be good enough (and super fast).

The jest+jsdom based test case should be something like:
```
  describe('mozCompare loaded as a tag script', () => {
    it('should be exposed as window.mozCompare', () => {
      const scriptEl = document.createElement('script');
      scriptEl.textContent = fs.readFileSync('./src/index.js'); 
      document.body.appendChild(scriptEl);
      expect(window.mozCompare('2', '1')).toEqual(1);
    });
  });
```

feel free to try it out and add it if you think that it would be a reasonable way to include that coverage with a minimal effort.

Reply:
that's right, thanks. I had updated the PR to only fix #2 in the meantime :)